src/EFCore.PG/Scaffolding/Internal/NpgsqlDatabaseModelFactory.cs

```diff
@@ -215,15 +207,16 @@ public override DatabaseModel Create(DbConnection dbConnection, DatabaseModelFac
             {
                 while (reader.Read())
                 {
-                    var table = new DatabaseTable
+                    var schema = reader.GetValueOrDefault<string>("nspname");
```

Comment:
So you're using extensions here. I have fixed all methods on the command to return an `NpgsqlDataReader`.

Reply:
Yeah. Any reason to do any change?

Reply:
See above.